### ISSUE TITLE
Deprecation warning when using ipaddr and other moved netcommon filters

### ISSUE THREAD

**Issue by piotrp** [ansible/ansible#77971]
### Summary

Using `ipaddr` and other moved netcommon filters, without FQCN, prints deprecation warnings in console.

It's because ansible_builtin_runtime.yml redirects it to ansible.netcommon, which in turn adds deprecation warning and redirects it to ansible.utils. 

Tested with basic installation of Ansiblr 6.0.0b2:

```sh
python -m venv .venv2
. ./.venv2/bin/activate
pip install 'ansible == 6.0.0b2' netaddr
ansible localhost -m debug -a 'msg={{ "127.0.0.1" | ipaddr }}'
```

### Issue Type

Bug Report

### Component Name

ipaddr

### Ansible Version

```console
$ ansible --version
ansible [core 2.13.0]
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/home/ppr/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /e/tmp/ansible-test/.venv2/lib/python3.8/site-packages/ansible
  ansible collection location = /home/ppr/.ansible/collections:/usr/share/ansible/collections
  executable location = /e/tmp/ansible-test/.venv2/bin/ansible
  python version = 3.8.10 (default, Mar 15 2022, 12:22:08) [GCC 9.4.0]
  jinja version = 3.1.2
  libyaml = True
```

### Configuration

```console
# if using a version older than ansible-core 2.12 you should omit the '-t all'
$ ansible-config dump --only-changed -t all
DEFAULT_REMOTE_USER(env: ANSIBLE_REMOTE_USER) = pprzybylski

BECOME:
======

CACHE:
=====

CALLBACK:
========

CLICONF:
=======

CONNECTION:
==========

paramiko_ssh:
____________
remote_user(env: ANSIBLE_REMOTE_USER) = pprzybylski

ssh:
___
remote_user(env: ANSIBLE_REMOTE_USER) = pprzybylski

HTTPAPI:
=======

INVENTORY:
=========

LOOKUP:
======

NETCONF:
=======

SHELL:
=====

VARS:
====
```

### OS / Environment

Ubuntu 20.04, Python venv with Ansible 6.0.0b2

### Steps to Reproduce

```sh
ansible localhost -m debug -a 'msg={{ "127.0.0.1" | ipaddr }}'
```

### Expected Results

No deprecation warning

### Actual Results

```console
[DEPRECATION WARNING]: Use 'ansible.utils.ipaddr' module instead. This feature will be removed from ansible.netcommon in a release after 2024-01-01. Deprecation warnings can be disabled by setting
deprecation_warnings=False in ansible.cfg.
localhost | SUCCESS => {
    "msg": "127.0.0.1"
}
```

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

**Comment by ansibot** [comment]
Files identified in the description:
* [`test/support/network-integration/collections/ansible_collections/ansible/netcommon/plugins/filter/ipaddr.py`](https://github.com/ansible/ansible/blob/devel/test/support/network-integration/collections/ansible_collections/ansible/netcommon/plugins/filter/ipaddr.py)

If these files are incorrect, please update the `component name` section of the description or use the `!component` bot command.

[click here for bot help](https://github.com/ansible/ansibullbot/blob/devel/ISSUE_HELP.md)

**Comment by stejoo** [comment]
Ran into the same deprecation warning on a box with Ansible 5.8.0 (2.12.5).
We (well @tonk mainly) figured this behaviour was caused by Ansible loading the `ansible.netcommon` collection before the `ansible.utils` collection. To remedy we added:
```
- hosts: ..
  collections:
    - ansible.utils
    - ansible.netcommon
  ..
```
to the Playbook, to reverse the order.

Apparently `ansible_builtin_runtime.yml` goes before collection "gathering" and it's order.
